File: app/src/main/java/vn/tiki/sample/database/AppDatabase.java
package vn.tiki.sample.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;
import vn.tiki.sample.entity.CartItem;

@Database(entities = { CartItem.class }, version = 1, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {
  public abstract CartDao cartDao();
}

File: sample/src/main/java/vn/tiki/sample/database/AppDatabase.kt
package vn.tiki.sample.database

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import vn.tiki.sample.entity.CartItem

@Database(entities = arrayOf(CartItem::class), version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
  abstract fun cartDao(): CartDao
}